Runtime/Core/Scripts/Operations/MetadataDownloader.cs

```diff
@@ -129,7 +129,17 @@ private static bool IsUpdateRequired(AvatarContext context)
                 return true;
             }
             AvatarMetadata previousMetadata = AvatarMetadata.LoadFromFile(context.AvatarUri.LocalMetadataPath);
-            return AvatarMetadata.IsUpdated(context.Metadata, previousMetadata);
+            return AvatarMetadata.IsUpdated(context.Metadata, previousMetadata) || ShouldUpdateAvatarFile(context);
+        }
+
+        private static bool ShouldUpdateAvatarFile(AvatarContext context)
+        {
+            if (!File.Exists(context.AvatarUri.LocalModelPath))
+            {
+                return true;
+            }
+            var avatarFileUpdateTime = File.GetLastWriteTime(context.AvatarUri.LocalModelPath);
```

Comment:
I think it is better to use UTC to prevent potential issues with local time. 
```cs
            var avatarFileUpdateTime = File.GetLastWriteTimeUtc(context.AvatarUri.LocalModelPath);
            return avatarFileUpdateTime < context.Metadata.UpdatedAt;
```